website/source/guides/secret-mgmt/db-root-rotation.html.md

```diff
@@ -0,0 +1,409 @@
+Key                Value
+---                -----
+lease_id           database/creds/readonly/999c43f0-f79e-ba90-24a8-4de5af33a2e9
+lease_duration     1h
+lease_renewable    true
+password           A1a-u7wxtrpx09xp40yq
+username           v-root-readonly-x6q809467q98yp4yx4z4-1525378026e
+
+
+# Make sure that you can connect to the database using the Vault generated credentials
+$ psql -h postgres.host.address -p 5432 \
+       -U v-root-readonly-x6q809467q98yp4yx4z4-1525378026e postgres
+Password for user v-root-readonly-x6q809467q98yp4yx4z4-1525378026:
+
+postgres=> \du
+Role name                                       |                         Attributes                         | Member of
+------------------------------------------------+------------------------------------------------------------+----------
+postgres                                        | Superuser, Create role, Create DB, Replication, Bypass RLS | {}
+v-root-readonly-x6q809467q98yp4yx4z4-1525378026 | Password valid until 2018-05-03 21:07:11+00                | {}
+
+postgres=> \q
+```
+
+
+#### API call using cURL
+
+```shell
+# Get a new set of credentials
+$ curl --header "X-Vault-Token: 1c97b03a-6098-31cf-9d8b-b404e52dcb4a" \
+       http://127.0.0.1:8200/v1/database/creds/readonly | jq
+{
+   "request_id": "527970fd-f5e8-4de5-d4ed-1b7970eaef0b",
+   "lease_id": "database/creds/readonly/ac79265e-668c-242f-4f67-1dae33da094c",
+   "renewable": true,
+   "lease_duration": 3600,
+   "data": {
+     "password": "A1a-0tr8u15y0us2u08v",
+     "username": "v-root-readonly-x7v65y1xuprzxv9vpt80-1525378873"
+   },
+   "wrap_info": null,
+   "warnings": null,
+   "auth": null
+}
+
+# Make sure that you can connect to the database using the Vault generated credentials
+$ psql -h postgres.host.address -p 5432 \
+       -U v-root-readonly-x6q809467q98yp4yx4z4-1525378026e postgres
+Password for user v-root-readonly-x6q809467q98yp4yx4z4-1525378026:
+
+postgres=> \du
+Role name                                       |                         Attributes                         | Member of
+------------------------------------------------+------------------------------------------------------------+----------
+postgres                                        | Superuser, Create role, Create DB, Replication, Bypass RLS | {}
+v-root-readonly-x6q809467q98yp4yx4z4-1525378026 | Password valid until 2018-05-03 21:07:11+00                | {}
+v-root-readonly-x7v65y1xuprzxv9vpt80-1525378873 | Password valid until 2018-05-03 21:21:18+00                | {}
+
+postgres=> \q
+```
+<br>
+
+This confirms that the Vault successfully connected to your PostgreSQL server
+and created a new user based on the privilege defined by `readonly.sql`.
+
+> The user credentials generated by the Vault has a limited TTL based on your
+configuration (`default_ttl`). In addition, you can revoke them if you suspect
+that it was compromised.
+
+**NOTE:** The `pg_hba.conf` is configured to prompt for password to
```

Comment:
I think it is ok to show the psql commands above to verify the config but advanced configuration like this doesn't feel right in our guides.

Reply:
I debated over this myself, so I'll remove it.   BTW, is there a better way to test/verify the feature?  I used the `psql` to test because I didn't know any other way.  But if you have a more elegant way to show that it worked, let me know.